docs/src/devdocs/mapping.md

```diff
@@ -0,0 +1,103 @@
+# Mapping of finite elements
+The shape functions and gradients stored in an `FEValues` object, is reinitialized for each cell by calling the `reinit!` function. The main part of this calculation, considers how to map the functions described on the reference cell, to the actual cell.
```

Comment:
Maybe this fits more nicely as "topic guide", not sure :D Very nice addition, regardless.

Reply:
I think I would leave it as devdocs for now, as it is the internal API.

Reply:
It doesn't really describe the API, just the mathematics.